index.html

```diff
@@ -0,0 +1,256 @@
+            Its tools are developed and supported by a community of responsive and passionate software developers and researchers.
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="special_section">
+      <div class="section_header">
+        Made With Manubot
+      </div>
+      <div class="row flex_wrap">
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+      </div>
+      <div class="row flex_wrap">
+        <a href="" class="button_link">
+          <i class="fas fa-book-open fa-lg"></i> Full Catalog
+        </a>
+      </div>
+    </section>
+    <section>
+      <div class="section_header">
+        Our Philosophy
+      </div>
+      <div class="row">
+        Manuscripts are the primary basis of scholarly communication, yet the process of creating them has not kept in step with the advancement of technology or the evolving best-practices of science.<br />
+        <br />
+        Traditionally, authoring of manuscripts takes place in a very closed manner.
+        While most scientific material is eventually distributed to some kind of public medium, the process of developing and reviewing the material is usually done in private, contained to a handful of individuals.
+        The tools used are often outmoded, restrictive, and overall poorly-suited for the task of collaborative, scientific writing.<br />
+        <br />
+        We believe that this process is antiquated and must be reformed.
+        Good science depends upon many things that the conventional approach doesn't provide:
+        A diversity of input and perspectives.
+        Spontaneous and cross-discipline contributions.
+        Organic and wide-spread discovery and dissemination of knowledge.
+        The fostering of a free, global information commons.
+        A method of real-time, continuous science, performed entirely in the open.<br />
+        <br />
+        Current technology easily affords huge advantages over the old way of doing things, both in convenience and in the quality of the result.
+        Manubot is a solution to the past and a push towards a future ideal, where scientific writing is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge.
+      </div>
+    </section>
+    <section class="special_section">
+      <div class="section_header">
+        Cite Manubot
+      </div>
+      <div class="row">
+        <div>
+          To reference the Manubot project, you can cite the 
+          "<a href="https://github.com/greenelab/meta-review">
+            Meta Review
```

Comment:
Note there are spaces like `" Meta Review "`.

Consider an alternative phrasing that does not use the label Meta Review. My problem with Meta Review is that it is not extremely accurate or descriptive and has poor searchability. However, if you think it is useful to show the name of this manuscript here, then OKAY.

Reply:
The only reason i put it is to avoid confusion. The real name of the paper is one thing, but then we're navigating them to a page that says "meta-review" in big letters at the top, and in the url. 
Also, the name at the top of the readme is not the same as the paper title.

I think it's useful to have both.

Reply:
Got it. I didn't catch that the URL pointed to the [GitHub repo](https://github.com/greenelab/meta-review) as opposed to the HTML mansucript at https://greenelab.github.io/meta-review/. Usually, I'd just point to that later for citation as people should cite the output rather than the source IMO.